app/helpers/patient_helper.rb

```diff
@@ -78,87 +66,356 @@ def normalize_and_get_state_name(name)
-      'florida' => { offset: -4, observes_dst: true },
-      'georgia' => { offset: -4, observes_dst: true },
-      'guam' => { offset: 10, observes_dst: false },
-      'hawaii' => { offset: -10, observes_dst: false },
-      'idaho' => { offset: -6, observes_dst: true },
-      'illinois' => { offset: -5, observes_dst: true },
-      'indiana' => { offset: -4, observes_dst: true },
-      'iowa' => { offset: -5, observes_dst: true },
-      'kansas' => { offset: -5, observes_dst: true },
-      'kentucky' => { offset: -4, observes_dst: true },
-      'louisiana' => { offset: -5, observes_dst: true },
-      'maine' => { offset: -4, observes_dst: true },
-      'marshallislands' => { offset: 12, observes_dst: false },
-      'maryland' => { offset: -4, observes_dst: true },
-      'massachusetts' => { offset: -4, observes_dst: true },
-      'michigan' => { offset: -4, observes_dst: true },
-      'minnesota' => { offset: -5, observes_dst: true },
-      'mississippi' => { offset: -5, observes_dst: true },
-      'missouri' => { offset: -5, observes_dst: true },
-      'montana' => { offset: -6, observes_dst: true },
-      'nebraska' => { offset: -5, observes_dst: true },
-      'nevada' => { offset: -7, observes_dst: true },
-      'newhampshire' => { offset: -4, observes_dst: true },
-      'newjersey' => { offset: -4, observes_dst: true },
-      'newmexico' => { offset: -6, observes_dst: true },
-      'newyork' => { offset: -4, observes_dst: true },
-      'northcarolina' => { offset: -4, observes_dst: true },
-      'northdakota' => { offset: -5, observes_dst: true },
-      'northernmarianaislands' => { offset: 10, observes_dst: false },
-      'ohio' => { offset: -4, observes_dst: true },
-      'oklahoma' => { offset: -5, observes_dst: true },
-      'oregon' => { offset: -7, observes_dst: true },
-      'palau' => { offset: 9, observes_dst: false },
-      'pennsylvania' => { offset: -4, observes_dst: true },
-      'puertorico' => { offset: -4, observes_dst: false },
-      'rhodeisland' => { offset: -4, observes_dst: true },
-      'southcarolina' => { offset: -4, observes_dst: true },
-      'southdakota' => { offset: -5, observes_dst: true },
-      'tennessee' => { offset: -5, observes_dst: true },
-      'texas' => { offset: -5, observes_dst: true },
-      'utah' => { offset: -6, observes_dst: true },
-      'vermont' => { offset: -4, observes_dst: true },
-      'virginislands' => { offset: -4, observes_dst: false },
-      'virginia' => { offset: -4, observes_dst: true },
-      'washington' => { offset: -7, observes_dst: true },
-      'westvirginia' => { offset: -4, observes_dst: true },
-      'wisconsin' => { offset: -5, observes_dst: true },
-      'wyoming' => { offset: -6, observes_dst: true },
-      nil => { offset: -4, observes_dst: true },
-      '' => { offset: -4, observes_dst: true }
+  # Offsets are DST
+  # rubocop:disable Metrics/MethodLength
+  def states_with_time_zone_data
+    {
```

Comment:
Thoughts on moving this into a JSON in `lib` or something that we can just import here?

Reply:
We can't represent nil keys in JSON but we can in YAML so definitely possible. Should we move `state_names` to yml as well because it's generally the same?

Reply:
@kylietmo updated this and other feedback in the latest changes